src/go/k8s/controllers/redpanda/cluster_controller.go

```diff
@@ -88,8 +88,13 @@ func (r *ClusterReconciler) Reconcile(
 		return ctrl.Result{}, fmt.Errorf("unable to retrieve Cluster resource: %w", err)
 	}
 
-	headlessSvc := resources.NewHeadlessService(r.Client, &redpandaCluster, r.Scheme, log)
-	nodeportSvc := resources.NewNodePortService(r.Client, &redpandaCluster, r.Scheme, log)
+	ports := map[string]int{
```

Comment:
not sure the controller is the place to define these, having methods on cluster_types would be better

Reply:
I took care of this https://github.com/vectorizedio/redpanda/pull/1021/commits/bfb60e5a6e7c8884d2008ac7c4ef741c41736249